tests/unit/amo/pages/TestAddonReviewList.js

```diff
@@ -935,6 +935,29 @@ describe(__filename, () => {
       );
     });
 
+    it('dispatches fetchReviewPermissions on update', () => {
+      const addon = { ...fakeAddon };
+      const userId = 66432;
+
+      loadAddon(addon);
+      dispatchSignInActions({ store, userId });
+      const dispatchSpy = sinon.spy(store, 'dispatch');
+
+      const root = render();
+      dispatchSpy.resetHistory();
```

Comment:
👍 

You can try writing code in a test driven development (TDD) style but sometimes I find that annoying. Without doing that, I always comment out my fixes to make sure I see the tests fail. This helps catch false positives like the one you ran into.

Reply:
I got it!
Thank you so much 😃